nrf_security/doc/driver_config.rst

```diff
@@ -84,36 +85,73 @@ You can use the following Kconfig options for fine-grained control over which dr
    * The :ref:`nrf_security_drivers_cc3xx` is limited to AES key sizes of 128 bits on devices with Arm CryptoCell cc310.
 
 
-MAC configuration
-*****************
+Key derivation
+**************
+
+To enable key derivation support, set one or more of the following Kconfig options:
+
++-------------------+-------------------------------------------------------+
+| Key derivation    | Configuration option                                  |
++===================+=======================================================+
+| HKDF              | :kconfig:option:`CONFIG_PSA_WANT_ALG_HKDF`            |
++-------------------+-------------------------------------------------------+
+| TLS 1.2 PRF       | :kconfig:option:`CONFIG_PSA_WANT_ALG_TLS12_PRF`       |
++-------------------+-------------------------------------------------------+
+| TLS 1.2 PSK to MS | :kconfig:option:`CONFIG_PSA_WANT_ALG_TLS12_PSK_TO_MS` |
++-------------------+-------------------------------------------------------+
+
+
+Key derivation driver configurations
+====================================
+
+You can use the following Kconfig options for fine-grained control over which drivers provide key derivation support:
+
++-------------------+--------------------------+-----------------------------------------------------------------------+
+| Key derivation    | nrf_cc3xx driver support | nrf_oberon driver support                                             |
++===================+==========================+==========================================+============================+
+| HKDF              | Not supported            | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_HKDF_OBERON`            |
++-------------------+--------------------------+-----------------------------------------------------------------------+
+| TLS 1.2 PRF       | Not supported            | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_TLS12_PRF_OBERON`       |
++-------------------+--------------------------+-----------------------------------------------------------------------+
+| TLS 1.2 PSK to MS | Not supported            | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_TLS12_PSK_TO_MS_OBERON` |
++-------------------+--------------------------+-----------------------------------------------------------------------+
+
+.. note::
+   * If a key derivation algorithm is enabled and no PSA driver enables or supports it, :ref:`nrf_security_drivers_builtin` support is enabled and used.
```

Comment:
'Is enabled' would imply it happens automatically?

Reply:
This is correct